### PR TITLE
watcher: wormchain support

### PR DESCRIPTION
**Pull request by evan-gray**

### CODE REVIEW

File: watcher/src/watchers/CosmwasmWatcher.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export class CosmwasmWatcher extends Watcher {
   rpc: string | undefined;
   latestBlockHeight: number;
 
-  constructor(chain: CosmWasmChainName) {
+  constructor(chain: CosmWasmChainName | 'wormchain') {
```
Comment (panoel) [inline watcher/src/watchers/CosmwasmWatcher.ts:17]: Should wormchain be a CosmWasmChainName?

**Review by panoel — DISMISSED** [review]
So, it looks like standard cosmwasm watcher pretty much just worked?

File: watcher/src/watchers/CosmwasmWatcher.ts (hunk 1 above)
Comment (evan-gray) [reply watcher/src/watchers/CosmwasmWatcher.ts:17]: It should, but it doesn't appear to be in the latest version of the sdk. idk if I can make an issue straight from here to the other repo

**Comment by evan-gray** [comment]
> So, it looks like standard cosmwasm watcher pretty much just worked?

Almost, except that the responses weren't quite the same so I had to copy a bunch. We should see if we can go back and clean up the accessor difference later.

**Review by panoel — APPROVED** [review]

**Pull request merged**